config/Symphony.config

```diff
@@ -3,6 +3,7 @@
     "minimizeOnClose" : true,
     "launchOnStartup" : true,
     "alwaysOnTop" : false,
+    "whiteListURL": "*",
```

Comment:
Can we have the config name as "whitelistUrl" instead of "whiteListURL" as it's a single word and to continue following the same standards as the other config names.